### PR TITLE
Refined steps to install Forseti on GCE

### LINKED ISSUE #3710: Refine Forseti on-GCE installation guide
Refine [Forseti on-GCE installation guide](https://forsetisecurity.org/docs/latest/setup/install.html) in an effort to improve website documentation.

### PR DESCRIPTION
**Pull request by red2k18**
Resolves #3710

### CODE REVIEW

File: _docs/_latest/setup/install/roles-and-permissions.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: Service Account roles and permissions
```
Comment (gkowalski-google) [inline _docs/_latest/setup/install/roles-and-permissions.md]: Rename to Service Account roles and required APIs? This page only talks about roles (no permissions), and also includes the APIs.

File: _docs/_latest/setup/install/index.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+---
+title: Install Forseti on Google Compute Engine
```
[the remark this answers is not in the document]
Comment (red2k18) [reply _docs/_latest/setup/install/index.md]: Not sure if everyone is familiar with the acronym, although I'd like to believe so. Hence, I'm not using the acronym. Renamed the title though.

File: _docs/_latest/setup/install/index.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+order: 100
```
[the remark this answers is not in the document]
Comment (red2k18) [reply _docs/_latest/setup/install/index.md:3]: Good point! Keeping the install page will add to the overhead, and so deleted it.

File: _docs/_latest/setup/install/roles-and-permissions.md (hunk 1 above)
Comment (red2k18) [reply _docs/_latest/setup/install/roles-and-permissions.md]: Sounds good, renamed.

File: _includes/docs/latest/setup-script-credentials.md [hunk 4]
```diff
@@ -1,6 +1,5 @@
 Terraform uses an IAM Service Account to deploy and configure resources on behalf of the user.  The Service Account and required APIs can be setup automatically with a provided script on the 
 [Forseti Terraform Github repository](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/setup.sh). 
-The Service Account and required APIs can also be configured manually by reviewing the [Requirements](https://forsetisecurity.org/docs/latest/setup/install.html#requirements) to deploy Forseti with Terraform.
```
Comment (gkowalski-google) [inline _includes/docs/latest/setup-script-credentials.md:3]: The Forseti on GKE installation steps references these setup-script files, and now the manual steps to create the service account and enable required APIs will not show up on the GKE steps. Can we add that back in, perhaps just add back into the forseti on gke page?

Comment (red2k18) [reply _includes/docs/latest/setup-script-credentials.md:3]: Both Forseti on-GCE and Forseti on-GKE installation pages are referencing the `setup-script-credentials.md` which has the instructions. I have updated the setup script so manual steps to create the service account will show up.

**Pull request merged**